### ISSUE TITLE
Using wildcard "packageFile" option does not play well with .npmrc files

### ISSUE THREAD

**Issue by lllopo** [raineorshine/npm-check-updates#1173]
- [ X ] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [ X ] I am using the latest version of `npm-check-updates`
- [ X ] I am using `node >= 14`

---

## Steps to Reproduce
Run ``ncu --packageFile */package.json`` with several subfolders containing package.json and one of them having authorisation keys in a .npmrc file along the package.json in it's folder.

```
root - { run NCU here }
|
|------- folder 1
|            |----------package.json
|            |----------.npmrc
|
|------- folder 2
            |----------package.json
```

## Current Behavior
NCU runs fine, finds all package.json files and goes through them, BUT fails to authorize for packages that require the keys from the .npmrc file.

## Expected Behavior
Since the .npmrc is along the package.json - it should be used properly, just as in the case when NCU is ran within the same folder, otherwise the wildcard option for the "packageFile" is pointless.

**Comment by raineorshine** [comment]
Thanks for the detailed bug report!

Fixed and published in `v16.0.5`.

Hopefully 🤞

**Comment by lllopo** [comment]
Yup, seems to run fine with my setup. Thanks.
